analyzer/src/main/kotlin/PackageJsonUtils.kt

```diff
@@ -79,7 +84,15 @@ internal class PackageJsonUtils {
         }
 
         private fun isYarnWorkspaceRoot(definitionFile: File): Boolean {
-            return definitionFile.readValue<ObjectNode>()["workspaces"] != null
+            return try {
+                definitionFile.readValue<ObjectNode>()["workspaces"] != null
+            } catch(e: JsonProcessingException) {
+                e.showStackTrace()
+
+                log.error { "Could not parse '${definitionFile.invariantSeparatorsPath}': ${e.message}" }
```

Comment:
Is this really something to log with 'error' severity?
I would even question whether the should be something logged at all as an empty json file seems to be a valid case?

Reply:
It's not a random JSON file, but a `package.json` which should never be empty. I think we should log an error because a significant part of the result could be missing if we cannot parse it.

Reply:
no strong opinion on that, I'm with as it is now.